src/components/multilineBox/tests/MultilineBoxWithRemoveButton.spec.tsx

```diff
@@ -0,0 +1,165 @@
+import {shallowWithState, shallowWithStore} from 'enzyme-redux';
+import * as React from 'react';
+import {mockStore} from 'redux-test-utils';
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../../ReactVapor';
+import {StringListActions} from '../../../reusableState/customList/StringListActions';
+import {RTestUtils} from '../../../utils/tests/RTestUtils';
+import {Button, IButtonProps} from '../../button/Button';
+import {multilineBoxWithRemoveButton} from '../hoc/MultilineBoxWithRemoveButton';
+import {IMultilineSingleBoxProps, MultilineBox} from '../MultilineBox';
+
+describe('Multiline box with remove button', () => {
+    describe('<MultilineBoxWithRemoveButton/>', () => {
+
+        let store: mockStore<IReactVaporState>;
+
+        const id = 'multiline-box';
+        const DefaultMultilineBoxWithRemoveButton = _.compose(
+            multilineBoxWithRemoveButton(),
+        )(MultilineBox);
+
+        it('should mount without errors', () => {
+            expect(() => shallowWithState(
+                <DefaultMultilineBoxWithRemoveButton
+                    id={id}
+                    data={[]}
+                    renderBody={_.identity}
+                    defaultProps={{}}
+                />, {}).dive().dive().dive(),

```

Comment:
It would be nice to add a function to shallow with the props to avoid the triple dive everywhere